stack_definition.json

```diff
@@ -135,155 +139,243 @@
+					"name": "kms_endpoint_type",
+					"value": "private"
 				},
 				{
-					"name": "existing_secrets_manager_crn",
-					"value": "ref:../../inputs/existing_secrets_manager_crn"
+					"name": "existing_cos_instance_crn",
+					"value": "ref:../../members/2b%20-%20Security%20Service%20-%20Object%20storage/outputs/cos_instance_id"
 				},
 				{
-					"name": "service_plan",
-					"value": "ref:../../inputs/secret_manager_service_plan"
+					"name": "enable_platform_logs",
+					"value": "ref:../../inputs/enable_platform_logs_metrics"
+				},
+				{
+					"name": "enable_platform_metrics",
+					"value": "ref:../../inputs/enable_platform_logs_metrics"
 				}
 			],
-			"name": "2b - Security Service - Secret Manager",
-			"version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.ba032ffb-286c-4a00-ad79-ae9e09cea687-global"
+			"name": "3 - Security Service - Observability",
+			"version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.756a5374-5be7-49b4-97f4-99d3b4011b54-global"
 		},
 		{
 			"inputs": [
 				{
-					"name": "ibmcloud_api_key",
-					"value": ""
+					"name": "existing_cos_instance_crn",
+					"value": "ref:../../members/2b%20-%20Security%20Service%20-%20Object%20storage/outputs/cos_instance_id"
 				},
 				{
-					"name": "cos_region",
-					"value": "ref:../../inputs/region"
+					"name": "skip_cos_kms_auth_policy",
+					"value": true
 				},
 				{
-					"name": "scc_region",
-					"value": "ref:../../inputs/region"
+					"name": "kms_endpoint_url",
+					"value": "ref:../../members/2a%20-%20Security%20Service%20-%20Key%20Management/outputs/kp_private_endpoint"
 				},
 				{
-					"name": "resource_group_name",
-					"value": "ref:../1 - Account Infrastructure Base/outputs/audit_resource_group_name"
+					"name": "existing_kms_instance_crn",
+					"value": "ref:../../members/2a%20-%20Security%20Service%20-%20Key%20Management/outputs/kms_instance_crn"
 				},
 				{
-					"name": "existing_kms_instance_crn",
-					"value": "ref:../2a - Security Service - Key Management/outputs/kms_instance_crn"
+					"name": "kms_endpoint_type",
+					"value": "private"
+				},
+				{
+					"name": "resource_group_name",
+					"value": "ref:../../members/1%20-%20Account%20Infrastructure%20Base/outputs/audit_resource_group_name"
 				},
 				{
 					"name": "use_existing_resource_group",
 					"value": true
 				},
 				{
-					"name": "provision_scc_workload_protection",
-					"value": false
+					"name": "region",
+					"value": "ref:../../inputs/region"
+				},
+				{
+					"name": "prefix",
+					"value": "ref:../../inputs/prefix"
+				},
+				{
+					"name": "existing_monitoring_crn",
+					"value": "ref:../../members/3%20-%20Security%20Service%20-%20Observability/outputs/cloud_monitoring_crn"
 				}
 			],
-			"name": "2c - Security Service - Security Compliance Center",
-			"version_locator": "7a4d68b4-cf8b-40cd-a3d1-f49aff526eb3.9e036202-5495-4ea3-926b-3751450bbf54-global"
+			"name": "4 - Security Service - Event Notifications",
```

Comment:
same as other comment, this is not a "Security Service"

Reply:
The is the same as above to indicate it is part of security services not really anything to do with genai